src/main/pg/dyn_notch.c

```diff
@@ -30,7 +30,7 @@
 PG_REGISTER_WITH_RESET_TEMPLATE(dynNotchConfig_t, dynNotchConfig, PG_DYN_NOTCH_CONFIG, 0);
```

Comment:
Should we update version?

Reply:
That's probably a good idea (to force the "Apply defaults" popup on first connect after flash, even if you didn't select "Full chip erase").

@SupaflyFPV can you imcrement the PG version, please?
```suggestion
PG_REGISTER_WITH_RESET_TEMPLATE(dynNotchConfig_t, dynNotchConfig, PG_DYN_NOTCH_CONFIG, 1);
```

Reply:
We are changing PG versions also into the maintenance versions?

Reply:
no, this needs to be a 4.4 PR

Reply:
> no, this needs to be a 4.4 PR

I made that change already